app/main.py

```diff
@@ -1,34 +1,158 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = True
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: tuple, end: tuple) -> None:
+        self.start = start
+        self.end = end
+        self.is_drowned = False
+        self.decks = []
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+    def crate_decks(self, row: int, column: int) -> None:
```

Comment:
There's a typo in the method name `crate_decks`. It should be `create_decks` to accurately describe its purpose and avoid confusion.